protocol/protocol.tex

```diff
@@ -7966,7 +7966,8 @@
 ``leftmost''.)
 
 \introlist
-\defining{\shadHash}, defined as a double application of \shaHash, is used to hash \blockHeaders:
+\defining{\shadHash}, defined as a double application of \shaHash, is used to hash \blockHeaders,
+and version 1-4 \transactionIDs:
```

Comment:
SHA-256d doesn't hash txids themselves; it hashes (the byte encoding, as specified in section 7.1, of) v1-4 transactions to produce txids.

Reply:
Thanks, I applied @dconnolly's suggestion, and also added "serialized byte encodings"